.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,57 +1,131 @@
-6. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
+If this is your first time submitting a PR, please familiarise yourself with:
+
+  Contributor Guidelines 
+  https://git.k8s.io/community/contributors/guide#your-first-contribution
+
+  Developer Guide 
+  https://git.k8s.io/community/contributors/devel/development.md#development-guide
+
+The following are additional useful links for PRs:
+
+  Best Practices for Faster Reviews 
+  https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
+
+  Marking Unfinished PRs for Review or as a Work in Progress (WIP)
+  https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
+-->
+
+**Which is the sponsoring or primary SIG associated with this PR?**
+<!--
+Label the sponsoring or primary SIG associated with this PR.
+
+For reference on available SIGs, you can find more details at:
+https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md#labels-that-apply-to-all-repos-for-both-issues-and-prs
+
+Add only ONE sig here via a /sig-primary <name> entry.
 -->
+/sig-primary ???
+
+> DISCUSSION REQUIRED
+>
+> Should we use the above pattern and update the label prow plugin ? https://github.com/kubernetes/test-infra/tree/master/prow/plugins/label. Logic would have to ensure that there is only a single primary sig associated. This metadata can be leveraged by other processes outside of the release notes.
+> 
+> or, if this is ONLY relevant to release notes then, we can add here as a `` ```primary-sig`` code block.
 
 **What type of PR is this?**
-> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
+<!-- 
+Label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request. 
+
+For reference on required PR labels, you can find more details at:
+https://git.k8s.io/community/contributors/devel/sig-release/release.md#issuepr-kind-label
+
+Add only ONE of the following /kind <type> entries.
+
+/kind api-change
+/kind bug
+/kind cleanup
+/kind design
+/kind documentation
+/kind failing-test
+/kind feature
+/kind flake
+-->
+/kind ???
+
+> DISCUSSION REQUIRED
 >
-> /kind api-change
-> /kind bug
-> /kind cleanup
-> /kind design
-> /kind documentation
-> /kind failing-test
-> /kind feature
-> /kind flake
+> Should we include a kind type of `deprecation`? And should we then include a `` ```depracation-notes`` code block to provide specific detailed notes around the depracation.
+> 
+> Should we only allow a single kind to be specified? There are a number of examples where multiple kind values make sense.
 
 **What this PR does / why we need it**:
+<!--
+Provide a description on what this PR does and/or why it is required.
+-->
+Description here ...
```

Comment:
I feel like `*What this PR does / why we need it` is self-explanatory and additional comments are not needed for this one...

Reply:
Agree - I went back and forth on this one. I was fighting consistency vs redundancy in my head. I'll remove the comment here.